.github/SECURITY.md

```diff
@@ -97,7 +97,7 @@ do not apply to Airflow, or have a different severity than some generic scoring
 
 ### What happens after you report the issue ?
 
-The [Airflow Security Team](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#security-team) will get back to you after assessing the report. You will usually get
+The Airflow Security Team will get back to you after assessing the report. You will usually get
```

Comment:
Shall we link this here: https://github.com/apache/airflow/blob/main/contributing-docs/01_roles_in_airflow_project.rst#security-team

Reply:
I removed it deliberately. I realized it's kinda useless information for the security reporter (and this chapter is for them) to read how the team is composed and how it rotates. Adds no value in this particular context.